ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -235,7 +221,7 @@ tests:
         pod should start\| pod sysctls\| build volumes should mount given secrets
         and configmaps into the build pod
     workflow: openshift-e2e-aws-heterogeneous-day-0
-- as: ocp-e2e-serial-aws-ovn-heterogeneous
+- as: ocp-e2e-serial-aws-ovn-multi-x-ax
```

Comment:
[TEST SUITE]-[PLATFORM]-[PAYLOAD]-[DEPLOYMENT STYLE]-[CONTROL PLANE ARCH]-[COMPUTE NODE ARCHITECTURE(S)]

I feel like a lot of our names break convention. In this name:
test suite = ocp-e2e-serial
platform = aws
payload? deployment type? should there be an optional [] for network type? = ovn
payload = multi
ctlp = x
cmp = ax